src/admin/index.js

```diff
@@ -426,6 +426,82 @@ module.exports = ({
       .pop()
   }
 
+  /**
+   * @param {string} groupId
+   * @param {string[]]} topics
+   * @param {boolean} [resolveOffsets=false]
+   * @return {Promise}
+   */
+  const fetchConsumerGroupOffsets = async ({ groupId, topics, resolveOffsets = false }) => {
+    if (!groupId) {
+      throw new KafkaJSNonRetriableError(`Invalid groupId ${groupId}`)
+    }
+
+    const coordinator = await cluster.findGroupCoordinator({ groupId })
+
+    let consumerOffsets
+    if (topics.length) {
+      const topicsToFetch = Promise.all(
+        topics.map(async topic => {
+          const partitions = await findTopicPartitions(cluster, topic)
+          const partitionsToFetch = partitions.map(partition => ({ partition }))
+          return { topic, partitions: partitionsToFetch }
+        })
+      )
+      const { responses } = await coordinator.offsetFetch({
```

Comment:
Instead of  `let consumerOffsets`
consider writing const `{ responses: consumerOffsets } = await coordinator.offsetFetch(....`

Reply:
Thanks! It doesn't work though, got an `ReferenceError: consumerOffsets is not defined`

Reply:
Yeah since the intention is to assign to `consumerOffsets` in the outer scope, they are not equivalent operations.